src/Diagnostics/FxCop/SDL.Analyzer.Common/Core/DiagnosticCategory.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace SDL.Analyzer.Common
```

Comment:
SDL isn't going to mean anything outside of MS. A more descriptive term would be better.